core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/components/dot-edit-content-sidebar-workflow/dot-edit-content-sidebar-workflow.component.html

```diff
@@ -11,34 +13,88 @@
         data-testId="workflow-name">
         @if (isLoading) {
             <p-skeleton />
+        } @else if (noWorkflowSelectedYet) {
+            <a
+                href="#"
+                class="select-workflow-link"
+                data-testId="select-workflow-link"
+                (click)="$event.preventDefault(); showDialog()"
+                tabindex="0">
+                {{ 'Select Workflow' | dm }}
+            </a>
         } @else {
             {{ scheme?.name }}
-        }
-    </dd>
 
-    <dt class="workflow-column__title" id="step-title">{{ 'Step' | dm }}</dt>
-    <dd
-        class="workflow-column__description"
-        aria-labelledby="step-title"
-        data-testId="workflow-step">
-        @if (isLoading) {
-            <p-skeleton />
-        } @else {
-            {{ step?.name || 'New' | dm }}
+            <!-- While not step, is consider as New contentlet -->
+            @if ($showWorkflowDialogIcon()) {
+                <p-button
+                    (click)="showDialog()"
+                    pButton
+                    styleClass="p-button-link"
+                    icon="pi pi-replay"></p-button>
+            }
         }
     </dd>
 
-    @if (task) {
-        <dt class="workflow-column__title" id="assignee-title">{{ 'Assignee' | dm }}</dt>
+    @if (!noWorkflowSelectedYet) {
+        <dt class="workflow-column__title" id="step-title">{{ 'Step' | dm }}</dt>
```

Comment:
'Step' should be defined in the language file.

Reply:
It is already translated as Step